apps/o2-blocks/src/github-issue-template/editor.js

```diff
@@ -127,7 +121,7 @@ registerBlockType( 'a8c/github-issue-template', {
 			},
 		};
 
-		const isValid = title && userOrOrg && repo;
+		const isValid = userOrOrg && repo;
```

Comment:
It can also be `Boolean( userOrOrg && repo)` for a better read. It's generally a good practice IMO. For example, if `isValid` would be `0` and we'd use it as a condition in the render function, it would output `0` in the markup instead of nothing. Another example would be an explicit comparison, i.e. `true === isValid` - it will return false even though isValid is truthy.

Reply:
Good points, I'll wrap in `Boolean()` 👍🏻